views/islandora_usage_stats.views.inc

```diff
@@ -168,6 +237,100 @@ function islandora_usage_stats_views_data() {
       ),
     ),
   );
+  $data['islandora_usage_stats_object_ds_access_log'] = array(
+    'table' => array(
+      'group' => t('Islandora Usage Stats'),
+      'base' => array(
+        'field' => 'id',
+        'title' => t('Islandora Usage Stats Object Datastream Access'),
+        'help' => t('The log of Datastreams being visited.'),
```

Comment:
This title should specify that this is datastream **downloads** and not access. Also the help makes it seem like viewing a datastream, which (perhaps this is just me) I think of as doing when I view the object in Islandora 😄